bridgenode/config.go

```diff
@@ -42,14 +47,12 @@ var (
 		`Set a custom bridgenode datadir. Usage: "-bridgedir='path/to/directory"`)
 	forestTypeCmd = argCmd.String("forest", "disk",
 		`Set a forest type to use (cow, ram, disk, cache). Usage: "-forest=cow"`)
+	quitAfterCmd = argCmd.Int("quitafter", -1,
+		`quit generating proofs after the given block height. (meant for testing)`)
 	cowMaxCache = argCmd.Int("cowmaxcache", 4000,
 		`how much memory to use in MB for the copy-on-write forest`)
-	memTTLdb = argCmd.Bool("memttldb", false,
-		`keep a cache of the ttldb in memory.`)
-	allInMemTTLdb = argCmd.Bool("allttldbinmem", false,
-		`keeps the entire ttldb in memory. Uses up a lot of memory.`)
-	quitAtCmd = argCmd.Int("quitat", -1,
-		`quit generating proofs after the given block height. (meant for testing)`)
+	memTTL = argCmd.Bool("memttl", false,
```

Comment:
If we're taking out the option for caching in ram, this flag should also be gone.